packages/server/src/fhir/operations/graphql.test.ts

```diff
@@ -670,7 +675,7 @@ describe('GraphQL', () => {
     `,
       });
     expect(res2.status).toBe(400);
-    expect(res2.body.issue[0].details.text).toEqual('Field "id" exceeds max depth (depth=14, max=12)');
+    expect(res2.body.issue[0].details.text).toEqual('Field "id" exceeds max depth (depth=13, max=12)');
```

Comment:
I assume this means our definition of "depth" changed, and before it was `14` and now it is `13`?

Does that mean we're implicitly allowing more depth?

Reply:
Or is the `13` just due to short circuiting?

```
// Short circuit, no need to keep computing
```